modules/telestion-api/src/main/java/de/wuespace/telestion/api/verticle/TelestionVerticle.java

```diff
@@ -60,6 +62,32 @@
 		}
 	}
 
+	/**
+	 * Creates a new Telestion verticle and tries to load the default configuration
+	 * from the specified configuration class.
+	 *
+	 * @param skipDefaultConfigLoading when {@code true} the loading of the default configuration is skipped
+	 */
+	public TelestionVerticle(boolean skipDefaultConfigLoading) {
+		if (skipDefaultConfigLoading) return;
+		try {
+			var defaultConfig = getConfigType().getConstructor().newInstance();
+			this.defaultConfig = defaultConfig;
+			this.defaultGenericConfig = defaultConfig.json();
+		} catch (NoSuchMethodException | InvocationTargetException | InstantiationException | IllegalAccessException e) {
+			// no default configuration on configuration class found, ignoring
+			logger.debug("No default configuration found. Skip loading");
```

Comment:
Let's make this message a little more expressive, something along the lines of 

> No default configuration found in configuration class `...`. Expected default constructor with no arguments to exist on `ConfigurationRecordClass`. Continuing without default configuration.

Reply:
Good point!
Done in [`0faeb44` (#466)](https://github.com/wuespace/telestion-core/pull/466/commits/0faeb448b63515961a118203ef2bf17870c204c2).

I also changed the log type to `info`. I think that makes more sense.